lms/views/admin/application_instance.py

```diff
@@ -312,6 +312,7 @@ def update_instance(self):
             ("microsoft_onedrive", "files_enabled", bool),
             ("vitalsource", "enabled", bool),
             ("vitalsource", "user_lti_param", str),
+            ("vitalsource", "user_lti_pattern", str),
```

Comment:
I think we should try `compile_user_lti_pattern` at some point in this view in a follow up PR.

Reply:
Yeah, that was the idea in making it public.